### DISCUSSION TITLE
Welcome to cim-compare Discussions!

### DISCUSSION THREAD

**Discussion by tviegut** [cimug-org/cim-compare#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ErkanUK** [comment]
To all interested in Python. I have created a python version and added a basic XML compare so it works with any LTDS or other XML file that is used to exchange CIM assets.
